src/assets/Enums.ts

```diff
@@ -267,7 +267,7 @@ export const TEST_TYPES_GROUP12_14: string[] = [
 
 // 39 - LEC with annual test for PSV, 45 - LEC without annual test for HGV, 44 - LEC with annual test for HGV
```

Comment:
This comment will need to be changed to be the new test types

Reply:
Good spot, comment has now been updated to reflect this.